packages/utils/src/get-awe-list.ts

```diff
@@ -0,0 +1,159 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { MetriportMedicalApi, PatientDTO } from "@metriport/api-sdk";
+import { executeAsynchronously } from "@metriport/core/util/concurrency";
+import { getEnvVar, getEnvVarOrFail } from "@metriport/core/util/env-var";
+import { out } from "@metriport/core/util/log";
+import { sleep } from "@metriport/core/util/sleep";
+import fs from "fs";
+import dayjs from "dayjs";
+import { Condition } from "@medplum/fhirtypes";
+
+const apiKey = getEnvVarOrFail("API_KEY");
+const apiUrl = getEnvVarOrFail("API_URL");
+const delayTime = parseInt(getEnvVar("BULK_INSERT_DELAY_TIME") ?? "200");
+const numberOfParallelExecutions = parseInt(getEnvVar("PARALLEL_QUERIES") ?? "10");
+const csvHeader = "id,firstName,lastName,dob,genderAtBirth,latestAWEDate,aweInPastYear\n";
+const curDateTime = new Date();
+const runName = `Z00s-${curDateTime.toISOString()}`;
```

Comment:
Can we expose the code `Z00` to a const so we can more easily change it?

Also, does it make sense to turn this script into something more generic, to search codes instead of specifically AWEs?